script/upload_to_rpm_repo

```diff
@@ -14,32 +14,37 @@ set -exu
 #
 # Date          Channel       Fingerprint           Full Key
 #
+# 10/17/2019    Release       0xA8580BDC82D3DC6C    9BD198A2D848C482E3550697A8580BDC82D3DC6C
 # 06/26/2019    Release       0x4FE13824E3FFC656    E85FFA8E2E90B40B33ED39274FE13824E3FFC656
 # 12/29/2018    Beta          0x0B31DBA06A8A26F9    9228DBCE20DDE5EC46488DE90B31DBA06A8A26F9
 # 12/29/2018    Dev           0x0B31DBA06A8A26F9    9228DBCE20DDE5EC46488DE90B31DBA06A8A26F9
 # 12/29/2018    Nightly       0x0B31DBA06A8A26F9    9228DBCE20DDE5EC46488DE90B31DBA06A8A26F9
 
-BUCKET=${1:-brave-browser-rpm-staging-nightly}
-GPG_KEY_ID=${2:-D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821} # Defaults to New Brave Software signing key
+BUCKET=${1:-brave-browser-rpm-staging-release-test}
+GPG_KEY_ID=${2:-9BD198A2D848C482E3550697A8580BDC82D3DC6C} # Defaults to New Brave Software signing key
 GPG_KEY_SHORT_ID=${GPG_KEY_ID:(-8)}
 
-if [ -z "${CHANNEL}" ]; then
-    echo "Error: Environment variable CHANNEL not set! Exitting..."
-    exit 1
-fi
-
 if [[ $2 == 0x* ]]; then
-    echo "Error: Please pass the Full GPG KEY ID to sign with! (i.e. 'E85FFA8E2E90B40B33ED39274FE13824E3FFC656')"
+    echo "Error: Please pass the Full GPG KEY ID to sign with! (i.e. '9BD198A2D848C482E3550697A8580BDC82D3DC6C')"
```

Comment:
We could make this a truncated string like "12AB34...C8D9E" so that the comment never looks outdated and that we don't copy/paste by mistake from the error message.

Reply:
Or even remove the entire key and just say "Error: Please pass the Full GPG KEY ID to sign with!" We're only talking about an error message in a script that is used in an automated process, so is not often if ever run manually these days.

Reply:
Resolved by removing any reference to an example key